static/skin/index.js

```diff
@@ -285,6 +297,7 @@
                         window.history.pushState({}, null, `${window.location.href.split('?')[0]}?lang=`);
                         setCookie(filterCookieName, 'lang=');
                         resetAndFilter();
+                        removeTagElement(true);
```

Comment:
>  Maybe the click event handler of the newly created div element deserves a separate named function too. This should be done in a separate refactoring commit.

Went against this because I feel the click handling is very much only for this div. Will change it to a named function if you do not like though :)

Reply:
It looks like the first part of my comment was not addressed.

Reply:
Sorry, that was a false observation. Got lost between the old and new version of this PR.

Reply:
> > Maybe the click event handler of the newly created div element deserves a separate named function too. This should be done in a separate refactoring commit.
> 
> Went against this because I feel the click handling is very much only for this div. Will change it to a named function if you do not like though :)

The utility of named functions is that - when named correctly - they help to understand the logic of the code easier without having to decipher it.

But now I have another question regarding the click handler - why do we need it at all? Won't the browser's following that link have the same effect?